mavenRepo/org/robobinding/aspectj-plugin/0.8.4/aspectj-plugin-0.8.4.jar.md5

```diff
@@ -1 +1 @@
-0d2d30bffed5714a3f623be3ac521e63
```

Comment:
Yeah, I don't wanna change 0.8.4 anything. Maybe my words make this complicated.

0.8.5 only works well with android plugin 2.+. Do your prefer this feature?